utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -620,6 +621,9 @@ def create_argument_parser():
     option('--playgroundsupport', store_true('build_playgroundsupport'),
            help='build PlaygroundSupport')
 
+    option('--pythonkit', store_true('build_pythonkit'),
```

Comment:
You'll likely need to add this to the test suite in `utils/build_swift/tests/expected_options.py`.